scripts.v3/utils.js

```diff
@@ -418,13 +406,43 @@ class ImporterExporter {
             }
 
             await this.httpClient.sendRequest("PUT", url, body);
-        }
-        catch (error) {
+        } catch (error) {
             throw new Error(`Unable to schedule website publishing. ${error.message}`);
         }
     }
+
+    /**
+     * Updates the url links of specified API Management service into updated url in destination.
+     */
+    async updateContentUrl(existingUrls, replaceableUrls) {
```

Comment:
`async updateContentUrl(existingUrls, replaceableUrls)`
Since it depends on matching the number of exiting and replaceable URLs, it would be great to have a check for that and respective error.

Reply:
Added a validation with description